src/libs/actions/Welcome.ts

```diff
@@ -147,7 +159,7 @@ function show({routes, showCreateMenu = () => {}, showPopoverMenu = () => {}}) {
 
         // If user is not already an admin of a free policy and we are not navigating them to their workspace or creating a new workspace via workspace/new then
         // we will show the create menu.
-        if (!Policy.isAdminOfFreePolicy(allPolicies) && !isDisplayingWorkspaceRoute) {
+        if (allPolicies && !Policy.isAdminOfFreePolicy(allPolicies) && !isDisplayingWorkspaceRoute) {
```

Comment:
why this logic change?

Reply:
did you check that `Policy.isAdminOfFreePolicy` returns `false` if `allPolicies` are `undefined`?